### DISCUSSION TITLE
Hornetnod and firefly connect

### DISCUSSION THREAD

**Discussion by Kireto80** [iotaledger/firefly#2615] [Help & Support]
Hi, I try to add hornetnod instalen in my mashine and give me this error:i type my lockal IP.
Wed, 23 Mar 2022 16:34:37 GMT
ClientError: `error sending request for url (https://192.168.1.109/api/v1/info): error trying to connect: invalid dnsname`

**Comment by MiKeF05** [comment]
You need to use a public domain for your node. Otherwise you can't connect via Firefly.

**Reply by Kireto80** [reply]
![Screenshot_20220325_150425_com anydesk anydeskandroid](https://user-images.githubusercontent.com/102247143/160126843-cf0393b8-0ed6-4747-98f8-074d876adb54.jpg)
What port need redirect?
